### DISCUSSION TITLE
Collecting stdout/result from job container

### DISCUSSION THREAD

**Discussion by ahoneycutt** [dagster-io/dagster#18399] [Q&A] [answered]
Hi all, is there a way to get the result/stdout from a container command when run using execute_k8s_job? From what I can find, it doesn’t appear so. I wanted to check here before rolling out a datastore or similar solution.

**Comment by alangenfeld** [comment, marked as answer]
Not `execute_k8s_job`, but we are looking at adding some capabilities like this for dagster pipes 

https://docs.dagster.io/guides/dagster-pipes
https://docs.dagster.io/guides/dagster-pipes/kubernetes#integrating-kubernetes-with-dagster-pipes
https://github.com/dagster-io/dagster/pull/18388

**Comment by ahoneycutt** [comment]
@alangenfeld That definitely looks like an improved approach to what I had in mind with `execute_k8s_job`. Thanks for the response, I'll mark as answered :+1:
